### ISSUE TITLE
New lines and tabulation

### ISSUE THREAD

**Issue by miyconst** [handsoncode-apps/Simplified#1]
You are free to use whatever format you like in your projects, but please follow up official Starcounter format for shared libraries such as `Simplified`.

https://github.com/StarcounterSamples/Polyjuice/wiki/C%23-code-formatting

Otherwise it will end up in space/tab fight.

@warpech, correct me if I am wrong.

**Comment by warpech** [comment]
Good catch, but this is also something we can do when accepting a pull request.

@miyconst let's use this: https://github.com/blog/1184-contributing-guidelines

Can you add a link to https://github.com/StarcounterSamples/Polyjuice/wiki/C%23-code-formatting
 in Simplified CONTRIBUTING.md?

And then add a link to CONTRIBUTING.md in Simplified README.md.

**Comment by miyconst** [comment]
Sure. Just created an issue for myself.
